test/gtest/api_convbiasactiv.cpp

```diff
@@ -59,6 +59,7 @@ struct CBATestCase
     }
 };
 
+GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST(ConvBiasActivFwdTest);
```

Comment:
Why is this required?

Reply:
`GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST ` allows empty list of parameterized test which is the case for Navi3x. If I do not do this it was giving error saying 

`Please use INSTANTIATE_TEST_SUITE_P to instantiate the tests, or GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST if the parameter list can be intentionally empty.`